src/h5web/vis-packs/core/heatmap/hooks.ts

```diff
@@ -38,36 +31,3 @@ export function useTooltipFormatters(
       format('.3')(dataArray.get(ordinateToIndex(y), abscissaToIndex(x))),
   };
 }
-
-export function useImageSize(imageRatio: number | undefined): Size {
```

Comment:
Moved to `core/hooks` and renamed to `useVisSize`. I'm open to a better naming, especially since it kinda conflicts with `computeVisSize`, which we use for the `contain` layout (although we could rename that function instead).

Reply:
As discussed, we can rename `computeVisSize` in `computeCanvasSize`